msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml2/util/MzMLPeaksDecoder.java

```diff
@@ -167,8 +193,8 @@
       return new double[0];
     }
 
-    EnumSet<MzMLBinaryDataInfo.MzMLCompressionType> compressions =
-        MzMLCompressionsHelper.getCompressions(compression);
+    EnumSet<MzMLCompressionType> compressions =
```

Comment:
What I meant by my comment on Slack was that you could remove this EnumSet completely and simply have 
````
switch (compression) {
  case ...:
}
````